src/static/javascripts/components/cards.jsx

```diff
@@ -7,33 +7,50 @@ var TitleSection = React.createClass({
       );
     }
 });
-
+//Card is really messy
 var Card = React.createClass({
-    
+    /* an initial state called response, actually not entirely sure
+     * why we have it.. :) 
+    */
     getInitialState: function(){
         return {response: []};
     },
     
+    //handleSurveySubmit is called whenever a submit button is pushed
+    //it calls POST on /api/response sending a JSON of the survey data
+    //and on success calls the removeHandler which removes the 
+    //corresponding cards 
     handleSurveySubmit: function(survey){
+        console.log(survey);
         $.ajax({
-            url: "/api/response",
+            url: this.props.routes.response,
 			contentType: 'application/json',
             type: 'POST',
             data: JSON.stringify(survey),
-            success: function(data){}.bind(this),
+            success: function(data){
+                console.log("Post success");
+                this.props.removeHandler();
```

Comment:
So when an ajax POST succeeds it goes here, so ajax is saying that the POST method to the response endpoint is working correctly, it also calls the removeHandler I set up to delete a survey after submitting